core/domain/email_manager.py

```diff
@@ -35,6 +35,7 @@
 
 # Stub for logging.error(), so that it can be swapped out in tests.
 def log_new_error(*args, **kwargs):
+    """Logs the ERROR message."""
```

Comment:
`Logs an error message. (This is a stub for logging.error(), so that the latter can be swapped out in tests.)`

Reply:
Done, thanks!